src/client/components/RegisterTeaser/RegisterTeaser.stories.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import RegisterTeaser from './RegisterTeaser';
+
+export default {
+  title: 'Components/Register Teaser',
+  component: RegisterTeaser,
+};
+
+export const Teaser = () => <RegisterTeaser singInURL="#" />;
```

Comment:
Please change the spelling here - signInURL

Reply:
thank you for pointing out ,I have fixed it everywhere.